### DISCUSSION TITLE
Connecting multiple GoPro to Mac via USB

### DISCUSSION THREAD

**Discussion by alexmomotiuk** [gopro/labs#817] [Q&A] [answered]
Hi, I'm developing macOS application where I need to show Webcam streams from two GoPro (connected via USB) simultaniously. Is it possible or not?

**Comment by dnewman-gpsw** [comment, marked as answer]
It is not a Labs question (custom firmware to modify camera features.) You are looking for Open GoPro for the SDK to provide camera control via wired or wireless connection. https://gopro.com/en/us/info/open-gopro

Yes, it sounds possible.
